scripts/install

```diff
@@ -176,7 +175,13 @@ download_binary() {
 		GVM_ARCH="386"
 	fi
 
-	GO_BINARY_FILE=${VERSION}.${GVM_OS}-${GVM_ARCH}${GVM_OS_VERSION}.tar.gz
+	extract_version $VERSION
+	compare_version $SEMVER "1.4.3"
```

Comment:
@dave-tucker, your patch looks really good, but this worries me... Having a new variable pops from nowhere (I see you set it inside extract_version) may be confusing, as it's just out of context in this file/function. I'd rather suggest you to modify extract_version in a way that it outputs its content and then set SEMVER here with the result of extract_version execution...
```bash
SEMVER="$(extract_version $VERSION)"
```
I'm pointing that out also, because I'm working on a fix for the Travis CI test error that's happening on macOS, and I could use your go version compare code in a cleaner way for this with this change.